lib/commands/setup.js

```diff
@@ -141,6 +141,10 @@ class SetupCommand extends Command {
 
                 const confirmed = await this.ui.confirm(`Do you wish to set up ${name}?`, true);
                 if (!confirmed) {
+                    if (step.onUserSkip) {
+                        step.onUserSkip(ctx);
```

Comment:
do we want to add an await here just to support any promise-based skips?

Reply:
Sure